test/Microsoft.ML.TimeSeries.Tests/TimeSeriesStaticTests.cs

```diff
@@ -0,0 +1,121 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Runtime.Api;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.StaticPipe;
+using System.Collections.Generic;
+using Xunit;
+
+namespace Microsoft.ML.Tests
+{
+    public sealed class TimeSeriesStaticTests
+    {
+
+        private sealed class Prediction
+        {
+            // Note that this field must be named "Data"; we ultimately convert
```

Comment:
> must be named "Data" [](start = 35, length = 21)

would help to add why this is the case

Reply:
The following lines of the comment were an attempt to explain why - is there a way I can explain this better?  

The big picture is that eventually we want to look at the result as an enumerable, and since that isn't supported natively in the static API, we convert the resulting dataview to dynamic.  Once we have an enumerable in the dynamic API, it requires string-named columns.  In the conversion of the static dataview to the dynamic dataview, ML.NET by default names the output column "Data", thus the user must use the field name Data here.  It wouldn't be hard to add a parameter to the static extension that allows the user to pick their own name, but @tomfinley was explaining that the static extensions were designed so user's did not need to use custom strings.

---
In reply to: [236471499](https://github.com/dotnet/machinelearning/pull/1656#discussion_r236471499) [](ancestors = 236471499)